tests/manifold/projection_manifold_01.cc

```diff
@@ -0,0 +1,74 @@
+//-------------------------------------------------------------------
+//    Copyright (C) 2016 by the deal.II authors.
```

Comment:
I found both versions of the text in the tests directory... If you think I should change it, I will...